Packs/Cymulate/Layouts/layoutscontainer-Cymulate_Immediate_Threats.json

```diff
@@ -0,0 +1,341 @@
+{
+    "id": "Cymulate Immediate Threats",
+    "group": "incident",
+    "name": "Cymulate Immediate Threat",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
+        "tabs": [
+            {
```

Comment:
Not critical, but since this is from 6.0 only
I would remove the legacy summary